model/validation.go

```diff
@@ -49,7 +49,8 @@ func IsDNS1123Label(value string) bool {
 	return len(value) <= dns1123LabelMaxLength && dns1123LabelRex.MatchString(value)
 }
 
-func validatePort(port int) error {
+// ValidatePort ...
+func ValidatePort(port int) error {
```

Comment:
Yeah, good point. I borrowed this style from k8s ;) I think this `Validate` should just be an interface which explains why comments feel dummy; too bad we can't define them easily for proto-generated sources.